src/interface/meta.thrift

```diff
@@ -77,6 +89,22 @@ struct EdgeItem {
     4: common.Schema        schema,
 }
 
+
+
+struct UserItem {
+    1: string account,
+    2: string first_name,
+    3: string last_name,
+    4: string email,
+    5: string phone,
```

Comment:
why supported `phone` ?

Reply:
> why supported `phone` ?

It's just a user property. It is optional. More such options may be added in the future.

Reply:
marked as `optional` will be better ?

Reply:
I think It is safe if set the optional to default (i.e. remove the optional keyword). What are the advantages of setting this parameter to `optional`?

Reply:
I don't think these attributes are necessary.

Reply:
optional and default are not the same. 
default means the field must be written out, but we could read it or not as needs.  If the filed is not set,  we will read a default value for it. 
optional means if the field is not set,  we will read a nullptr.

In this case,  default and optional both are ok.  All of them are backward compatible. 
But for email and phone,  i think we have no needs for them.  But there are some other options we could support.   See https://dev.mysql.com/doc/refman/8.0/en/create-user.html